protos/orchestrator_service.proto

```diff
@@ -318,6 +318,7 @@ message OrchestratorRequest {
     repeated HistoryEvent newEvents = 4;
     OrchestratorEntityParameters entityParameters = 5;
     bool requiresHistoryStreaming = 6;
+    map<string, google.protobuf.Value> orchestratorContextConfig = 7;
```

Comment:
Let's call this simply `properties` or `customProperties`. That's more in-line with the convention for adding user-defined metadata to requests in various frameworks.